framework/core/hpp_instance.cpp

```diff
@@ -21,11 +21,23 @@
 #include <core/hpp_physical_device.h>
 #include <volk.h>
 
+#if defined(VKB_DEBUG) || defined(VKB_VALIDATION_LAYERS) || (defined(VKB_VALIDATION_LAYERS_GPU_ASSISTED) || defined(VKB_VALIDATION_LAYERS_BEST_PRACTICES))
```

Comment:
With #737, the VKB_VALIDATION_LAYERS_* implicitly enable VKB_VALIDATION_LAYERS.
So this condition (here and in instance.cpp) could be simplified. (Otherwise, it would be missing the check on VKB_VALIDATION_LAYERS_SYNCHRONIZATION)

Reply:
Exactly. There is no need to check for the additional validation flags here.

Reply:
Sorry just got back onto samples. Fixed this. Thank you for the suggestion, forgot that the layers are now implicitly enabled